packages/babel-plugin-polyfill-corejs3/src/index.ts

```diff
@@ -300,12 +294,12 @@ export default defineProvider<Options>(function (
           resolved.desc,
           `${resolved.name}InstanceProperty`,
           utils,
-          // $FlowIgnore
+          // @ts-expect-error
           meta.object,
         );
         if (!id) return;
 
-        if (isCall) {
+        if (path.parentPath.isCallExpression({ callee: path.node })) {
           callMethod(path, id);
         } else {
           path.replaceWith(t.callExpression(id, [path.node.object]));
```

Comment:
This is the _very last_ type error currently -- I assume it's because TS isn't able to properly narrow on `else`s or something